website/docs/r/group_member.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "azuread"
+page_title: "Azure Active Directory: azuread_group_member"
+sidebar_current: "docs-azuread-resource-azuread-group-member"
+description: |-
+  Manages a Group Membership within Azure Active Directory.
+
+---
+
+# azuread_group_member
+
+Manages a Group Membership within Azure Active Directory.
```

Comment:
Could we add a warning here to not use this resource and the `azuread_group.members` property at the same time?